templates/markdown/.partials/tags.md

```diff
@@ -1,7 +1,5 @@
-<div class="tags">
+
 {{#each tags}}
-  <div class="tags__tag">{{./name}}</div>
-{{else}}
-  <div class="tags__no-tags">No tags</div>
```

Comment:
Note to this change: the else-condition is never reached as all template includes that reference this file have a conditional that checks for tags in-place (something like `if x.tags ...`).